site/app/views/calendar/CalendarView.php

```diff
@@ -59,7 +59,8 @@ public function showCalendar(CalendarInfo $info, bool $in_course = false): strin
             "empty_message" => $info->getEmptyMessage(),
             "in_course" => $in_course,
             "is_instructor" => $this->core->getUser()->getGroup() === User::GROUP_INSTRUCTOR,
-            "colors" => $info->getColors()
+            "colors" => $info->getColors(),
+            "view_cookie" => isset($_COOKIE['view']) ? $_COOKIE['view'] : "view=month"
```

Comment:
Why is the string we're passing around here `view=month`?  Would it not be more appropriate to just pass `month`?

Reply:
I passed it as "view=month" since if the cookie doesn't exist, then I pass in a string instead of a cookie.

Reply:
it would!